tests/fixtures/test_vectors.json

```diff
@@ -0,0 +1,132 @@
+{
+    "general": {
+        "SIGNER_TYPE": "Stronghold",
+        "MNEMNONIC": "acoustic trophy damage hint search taste love bicycle foster cradle brown govern endless depend situate athlete pudding blame question genius transfer van random vast",
+        "MNEMNONIC_ADDRESS": "iota1qpswqe4v8z2cdtgc7sfj0hfneqh37lhmjgnth36mfndwcxkjrakcvpmm727"
```

Comment:
This is the public address at index 1, maybe we should use another name to make this clear?

Reply:
Good suggestion. Modified.